java/pom.xml

```diff
@@ -208,7 +208,7 @@
             <configuration>
               <rules>
                 <requireMavenVersion>
-                  <version>[3.0.4,4)</version>
+                  <version>[3.3.0,4)</version>
```

Comment:
why do we have to specify an upper bound?

Reply:
Probably there are API changes in Maven 4. So, when you accidentally do a release with this new version, there might be some unexpected behavior.

Reply:
Thank you for the clarification.